ios/PlayKitOVPStarter/PlayKitOVPStarter/ViewController.swift

```diff
@@ -1,30 +1,32 @@
-import UIKit
 import PlayKit
-import PlayKitUtils
 import PlayKitKava
 import PlayKitProviders
+import PlayKitUtils
 import StreamrootSDK
+import UIKit
 
 // VOD
-fileprivate let SERVER_BASE_URL = "https://cdnapisec.kaltura.com"
-fileprivate let PARTNER_ID = 1424501
-fileprivate let ENTRY_ID = "1_djnefl4e"
+private let SERVER_BASE_URL = "https://cdnapisec.kaltura.com"
+private let PARTNER_ID = 1424501
+private let ENTRY_ID = "1_djnefl4e"
 
 // Live DVR
-//fileprivate let SERVER_BASE_URL = "https://cdnapisec.kaltura.com"
-//fileprivate let PARTNER_ID = 1740481
-//fileprivate let ENTRY_ID = "1_fdv46dba"
+// fileprivate let SERVER_BASE_URL = "https://cdnapisec.kaltura.com"
```

Comment:
maybe we can delete those ids? I don't know if its belong to kaltura

Reply:
It does. Ditto with my previous comment